templates/directory.php

```diff
@@ -274,13 +274,14 @@ function pmpromd_shortcode($atts, $content=null, $code="")
 					{
 						$auser = get_userdata($auser->ID);
 						$auser->membership_level = pmpro_getMembershipLevelForUser($auser->ID);
+						$user_identifier = apply_filters( 'pmpromd_user_identifier', 'user_nicename' );
```

Comment:
Let's add phpdoc info for this hook

Reply:
Actually, this is used in multiple places, let's make a new function that has the hook in that and call that function here.